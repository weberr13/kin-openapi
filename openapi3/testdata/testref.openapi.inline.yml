---
openapi: 3.0.0
info:
  title: 'OAI Specification w/ refs in YAML'
  version: '1'
paths: { }
components:
  schemas:
    AnotherTestSchema:
      type: object
      properties:
        remoteName:
          "$ref": components.openapi.yml#/components/schemas/Name
    Inline2:
      type: object
      properties:
        name:
          type: string
    Recursive:
      required:
        - directive
      properties:
        directive:
          type: string
          description: the name of the recursive shcema
        args:
          type: array
          description: Recursive schema arguments
          items:
            type: string
        block:
          type: array
          items:
            $ref: "#/components/schemas/Recursive"
