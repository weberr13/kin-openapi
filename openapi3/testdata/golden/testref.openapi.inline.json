{"components":{"schemas":{"AnotherTestSchema":{"properties":{"remoteName":{"$ref":"#/components/schemas/Name"}},"type":"object"},"Inline2":{"properties":{"name":{"type":"string"}},"type":"object"},"Name":{"type":"string"},"Recursive":{"properties":{"args":{"description":"Recursive schema arguments","items":{"type":"string"},"type":"array"},"block":{"items":{"$ref":"#/components/schemas/Recursive"},"type":"array"},"directive":{"description":"the name of the recursive shcema","type":"string"}},"required":["directive"]}}},"info":{"title":"OAI Specification w/ refs in YAML","version":"1"},"openapi":"3.0.0","paths":{}}